docs/appendices/release-notes/unreleased.rst

```diff
@@ -86,6 +86,8 @@ Changes
 
 - Added support of ``numeric`` type to the ``avg`` aggregation function.
 
+- Enabled session settings to be preserved across http requests.
+
```

Comment:
Maybe clarify that this is only the case within the same connection. So only if the client re-uses the conneciton.

Reply:
Thanks, I included those points. Please have another look.